packages/column-views/src/maps/column-navigation/mapbox/state.ts

```diff
@@ -26,7 +26,7 @@ export interface NavigationProviderProps {
   inProcess?: boolean;
   selectedColumn?: number | null;
   hoveredColumn?: number | null;
-  columns?: ColumnGeoJSONRecordWithID[] | null;
+  columns?: number[] | null;
```

Comment:
Don't change the behavior of an existing method! This could be done but would be a major breaking change.

Instead, either add a `number[]` option to the existing type or create a new `columnIDs` or `filteredColumnIDs` option